dvb-t/es-Almeria

```diff
@@ -4,7 +4,7 @@
 	FREQUENCY = 578000000
 	BANDWIDTH_HZ = 8000000
 	CODE_RATE_HP = 2/3
-	CODE_RATE_LP = NONE
+	CODE_RATE_LP = 2/3
```

Comment:
I don't think that this is right. Hierarchy is not set thus LP value is unused and should be really set to NONE. I fixed the autodiscovery in tvh to use NONE value when hierarchy is unused. v4.1-2112-gaf333d9